pkg/kubelet/apis/podresources/server_v1_test.go

```diff
@@ -674,14 +679,14 @@ func TestAllocatableResources(t *testing.T) {
 				CpuIds: allCPUs,
 			},
 		},
-		{
-			desc:       "no devices, no CPUs, all memory",
-			allCPUs:    []int64{},
-			allDevices: []*podresourcesapi.ContainerDevices{},
-			expectedAllocatableResourcesResponse: &podresourcesapi.AllocatableResourcesResponse{
-				Memory: allMemory,
-			},
-		},
+		// {
+		// 	desc:       "no devices, no CPUs, all memory",
+		// 	allCPUs:    []int64{},
+		// 	allDevices: []*podresourcesapi.ContainerDevices{},
+		// 	expectedAllocatableResourcesResponse: &podresourcesapi.AllocatableResourcesResponse{
+		// 		Memory: allMemory,
+		// 	},
+		// },
```

Comment:
the fix for the memory part is simple:
```
-               // {
-               //      desc:       "no devices, no CPUs, all memory",
-               //      allCPUs:    []int64{},
-               //      allDevices: []*podresourcesapi.ContainerDevices{},
-               //      expectedAllocatableResourcesResponse: &podresourcesapi.AllocatableResourcesResponse{
-               //              Memory: allMemory,
-               //      },
-               // },
+               {
+                       desc:       "no devices, no CPUs, all memory",
+                       allCPUs:    []int64{},
+                       allDevices: []*podresourcesapi.ContainerDevices{},
+                       allMemory:  allMemory,
+                       expectedAllocatableResourcesResponse: &podresourcesapi.AllocatableResourcesResponse{
+                               Memory: allMemory,
+                       },
+               },
```

equivalently, vs current master (without this PR):
```
$ git diff
diff --git a/pkg/kubelet/apis/podresources/server_v1_test.go b/pkg/kubelet/apis/podresources/server_v1_test.go
index aaa560fc4f0..17ae5c6c2a5 100644
--- a/pkg/kubelet/apis/podresources/server_v1_test.go
+++ b/pkg/kubelet/apis/podresources/server_v1_test.go
@@ -678,6 +678,7 @@ func TestAllocatableResources(t *testing.T) {
                        desc:       "no devices, no CPUs, all memory",
                        allCPUs:    []int64{},
                        allDevices: []*podresourcesapi.ContainerDevices{},
+                       allMemory:  allMemory,
                        expectedAllocatableResourcesResponse: &podresourcesapi.AllocatableResourcesResponse{
                                Memory: allMemory,
                        },
```